frontend/catalyst/utils/toml.py

```diff
@@ -300,6 +300,11 @@ def patch_schema1_collections(
             invertible=False, controllable=False, differentiable=False
         )
 
+    # TODO: remove after PR #642 is merged in lightning
+    # CPhase is an alias, which might exist in a toml schema 1 configs. We remove it here.
+    if 'CPhase' in matrix_decomp_props:
+        matrix_decomp_props.pop("CPhase")
```

Comment:
We're about to merge 642 so let's remove these 3 blocks with TODOs

Reply:
No more relevant